### ISSUE TITLE
Training Example

### ISSUE THREAD

**Issue by LL3RD** [okunator/cellseg_models.pytorch#58]
Thank you for your generous sharing！ Could you give more training examples about Hovernet model and Omnipose model, i'm confused about their default setting in this code, e.g. the loss defined in hovernet seems different from your setting.

**Comment by okunator** [comment]
Hi,

Thanks for the feedback. I'll work out more examples in the next release. You can use the same losses as in the HoVer-Net paper if you wish but the idea in this repo has been to write the code flexibly so that ppl could train the models as they wish. The loss defaults are pretty basic so you might want to adjust them to your needs. Also, in the HoVer-Net paper they used their own grad-MSE loss which was the just the MSE taken from the gradient of the gt-mask and the model prediction. I implemented it but learned pretty quickly that it blew up the gradients easily so I've just ignored it thus far. I might try to rewrite in the future..
